### PR TITLE
docs(auth): update "ActionCodeSettings" example code

### PR DESCRIPTION
**Pull request by yu1ro**
## Description

I think "Open link in app" in "Verifying a users email" section's example code is something wrong.
Isn't it the latest example code?

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by yu1ro** [comment]
I'm sorry my commit message seems to be not compatible.
I will change message.

**Comment by yu1ro** [comment]
@googlebot I signed it!

File: docs/auth/usage.mdx [hunk 1]
```diff
@@ -268,28 +268,15 @@ User? user = FirebaseAuth.instance.currentUser;
...
-        handleCodeInApp: true);
-
-await user.sendEmailVerification(actionCodeSettings);
+      url: 'https://www.example.com/?email=${user.email}',
+      dynamicLinkDomain: 'example.page.link',
+      androidPackageName: 'com.example.android',
+      androidInstallApp: true,
+      androidMinimumVersion: '12",
+      iOSBundleId: 'com.example.ios',
+      handleCodeInApp: true);
```
Comment (russellwheatley) [inline docs/auth/usage.mdx:277]:
```suggestion
      handleCodeInApp: true,
      );
```

**Review by russellwheatley — CHANGES_REQUESTED** [review]
small tweak but LGTM.

File: docs/auth/usage.mdx (hunk 1 above)
Comment (yu1ro) [reply docs/auth/usage.mdx]: Thank you for your review.
I applied it.

**Review by russellwheatley — APPROVED** [review]
LGTM

**Pull request merged**